### PR TITLE
Update Readme.md (Style Guide)

### PR DESCRIPTION
**Pull request by TaylorShiells**
This change would establish the Google Developer Documentation Style Guide as the official style guide for the ReadMe. This will provide a starting point for editors or writers looking to work on documentation. This edit also does an initial path on the readme based on that style guide

## Short roundup of the initial problem
The readme lacks a style guide. Editors have no starting point for improving the readme.

## What will change with this Pull Request?
- Establishes a Style Guide
- Edits the readme in accordance with the style guide

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +52,7 @@ We try to be very responsive to new issues. We'll try to give you advice on how
 
 # Translation Status [![Cockatrice on Transiflex](https://tx-assets.scdn5.secure.raxcdn.com/static/charts/images/tx-logo-micro.c5603f91c780.png)](https://www.transifex.com/projects/p/cockatrice/)
 
-Cockatrice uses Transifex for translations. You can help us bring Cockatrice and Oracle to your language or just edit single wordings right from within your browser by simply visiting our [Transifex project page](https://www.transifex.com/projects/p/cockatrice/).<br>
+Cockatrice uses Transifex for translations. You can help us bring Cockatrice and Oracle to your language or just edit single wordings right from within your browser by simly visiting our [Transifex project page](https://www.transifex.com/projects/p/cockatrice/).<br>
```
Comment (Daenyth) [inline README.md]: Typo here: `simly`

File: README.md [hunk 2]
```diff
@@ -78,14 +80,6 @@ To compile:
     cmake ..
     make
 
-You can then run
```
Comment (Daenyth) [inline README.md]: Why remove these instructions?

File: README.md [hunk 3]
```diff
@@ -2,43 +2,45 @@
 
 ---
 
-**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [License](#license-)
+**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [ReadMe Style Guide](#style-) | [License](#license-)
```
Comment (Daenyth) [inline README.md]: The style anchor is broken - I think it needs to be `#readme-`

Comment (skwerlman) [reply README.md]: Could the table of contents be split onto multiple lines? Right now it wraps at an odd place

File: README.md [hunk 4]
```diff
@@ -107,21 +101,23 @@ The following flags can be passed to `cmake`:
...
->Please note that running this command will expose the TCP port 4747 of the docker container<br>
+>Note: Running this command exposes the TCP port 4747 of the docker container<br>
 to permit connections to the server.
 
-More infos on how to use Servatrice with Docker can be found in our [wiki](https://github.com/Cockatrice/Cockatrice/wiki/Setting-up-Servatrice#using-docker).
+Find more information on how to use Servatrice with Docker in our [wiki](https://github.com/Cockatrice/Cockatrice/wiki/Setting-up-Servatrice#using-docker).
+
+# ReadMe Style Guide and Documentation Process
 
+This ReadMe uses the [Google Developer Documentation Style Guide] to ensure consistent writing across the ReadMe.
```
Comment (skwerlman) [inline README.md]: Should this be a link?

**Comment by skwerlman** [comment]
Is this style guide intended to apply only to docs in the source tree, or would it apply to the wiki as well?

File: README.md (hunk 2 above)
Comment (TaylorShiells) [reply README.md]: There was no intent to remove anything here, which instructions have you seen removed?

File: README.md (hunk 3 above)
Comment (TaylorShiells) [reply README.md]: Good catch, thanks

File: README.md (hunk 4 above)
Comment (TaylorShiells) [reply README.md]: Link added, thanks

**Comment by TaylorShiells** [comment]
@skwerlman Ideally the style guide would apply wherever it could. The goal here is to create a standard so that anyone interested in documentation can start publishing edits.

File: README.md [hunk 5]
```diff
@@ -2,43 +2,45 @@
...
+To contribute code to the project, please review [the guidelines](https://github.com/Cockatrice/Cockatrice/blob/master/.github/CONTRIBUTING.md).
+We maintain a tag for "easy" changes on our issue tracker. Issues with this tag provide a simple way to get started. [Issues tagged as Easy Changes](https://github.com/Cockatrice/Cockatrice/issues?q=is%3Aopen+is%3Aissue+label%3A%22Easy+Change%22)
 
-Check our long-term project **roadmap** with some visions [here](https://docs.google.com/document/d/1Ewe5uSaRE2nR2pNPMaGmP6gVZdqgFbBgwSscGqIr4W0/edit).
+Read the long-term project **roadmap** to see planned edits and milestones [here](https://docs.google.com/document/d/1Ewe5uSaRE2nR2pNPMaGmP6gVZdqgFbBgwSscGqIr4W0/edit).
 
-We try to be very responsive to new issues. We'll try to give you advice on how a feature should be implemented / advice on places the codebase is doing something similar before you get too far along with a PR.
+We try to be responsive to new issues. We'll provide advice on how best to implement a feature; alternately, we can show you where a contributor is already doing something similar in the codebase.
```
Comment (tooomm) [inline README.md]: >(...) alternately, we can show you where a contributor is already doing something similar in the codebase.

I'm not sure that's what the initial text tried to say.

File: README.md [hunk 6]
```diff
@@ -107,21 +101,23 @@ The following flags can be passed to `cmake`:
 
 #### Servatrice Docker container
 
-A Dockerfile is provided to run Servatrice (the Cockatrice server) using [Docker](https://www.docker.com/what-docker).<br>
+Use [Docker](https://www.docker.com/what-docker) to run Servatrice (the Cockatrice server) using the Cockatrice Dockerfile.<br>
```
Comment (tooomm) [inline README.md]: Does this only sounds weird to me?
~`Use Docker to run Servatrice using the Cockatrice Dockerfile.`

File: README.md [hunk 7]
```diff
@@ -2,43 +2,45 @@
 
 ---
 
-**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [License](#license-)
+**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [ReadMe Style Guide](#readme-) | [License](#license-)
```
Comment (tooomm) [inline README.md]: >The style anchor is broken - I think it needs to be `#readme-`

Isn't it even `#readme-style-guide-and-documentation-process`? :)
Couldn't verify if it's working with that change on your fork, I can't access the changed README file on your branch you pr'ed from somehow.

File: README.md (hunk 2 above)
Comment (tooomm) [reply README.md]: You removed this part:

>You can then run
>```
>make install
>```
>to get a cockatrice installation inside the `release` folder, or:
>``` 
>make package
>```

File: README.md [hunk 8]
```diff
@@ -107,21 +101,23 @@ The following flags can be passed to `cmake`:
...
->Please note that running this command will expose the TCP port 4747 of the docker container<br>
+>Note: Running this command exposes the TCP port 4747 of the docker container<br>
 to permit connections to the server.
 
-More infos on how to use Servatrice with Docker can be found in our [wiki](https://github.com/Cockatrice/Cockatrice/wiki/Setting-up-Servatrice#using-docker).
+Find more information on how to use Servatrice with Docker in our [wiki](https://github.com/Cockatrice/Cockatrice/wiki/Setting-up-Servatrice#using-docker).
+
+# ReadMe Style Guide and Documentation Process
 
+This ReadMe uses the [Google Developer Documentation Style Guide](https://developers.google.com/style/) to ensure consistent writing across the ReadMe.
```
Comment (tooomm) [inline README.md]: Why that huge headline?
The link isn't about `Documentation Process` too as far as I can tell.

Or do you want to add something about that topic here too?

<br>

We could also just use a bottom line at the very end instead a huge paragraph:
> This README uses the [Google Developer Documentation Style Guide](https://developers.google.com/style/) to ensure consistent writing(documentation?) across the project.

Same for CONTRIBUTING file or wiki main page!
Or even just a general hint here in readme that applies to all project.
`This project uses the ...`

File: README.md [hunk 9]
```diff
@@ -2,43 +2,45 @@
...
-**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [License](#license-)
+**Table of Contents** &nbsp;&nbsp; [Cockatrice](#cockatrice) | [Downloads](#downloads-) | [Get Involved](#get-involved-) | [Community](#community-resources) | [Translation](#translation-status-) | [Building](#building--) | [Running](#running) | [ReadMe Style Guide](#style-) | [License](#license-)
 
 ---
 
 <br><pre>
-<b>If you're getting started &#8674; [view our webpage](https://cockatrice.github.io/)</b><br>
-<b>If you're trying to get support or suggest changes &#8674; [file an issue](https://github.com/Cockatrice/Cockatrice/issues) ([How?](https://github.com/Cockatrice/Cockatrice/wiki/How-to-Create-a-GitHub-Ticket))</b>
+<b>To get started, &#8674; [view our webpage](https://cockatrice.github.io/)</b><br>
+<b>To get support or suggest changes &#8674; [file an issue](https://github.com/Cockatrice/Cockatrice/issues) ([How?](https://github.com/Cockatrice/Cockatrice/wiki/How-to-Create-a-GitHub-Ticket))</b>
```
[the remark this answers is not in the document]
Comment (TaylorShiells) [reply README.md]: I think that's a good idea.

File: README.md (hunk 5 above)
Comment (TaylorShiells) [reply README.md]: Right, I'll give that another pass.

File: README.md (hunk 8 above)
Comment (TaylorShiells) [reply README.md]: Merging it into an existing tab seems like a proper answer, I'll try to address this.

File: README.md (hunk 6 above)
Comment (ZeldaZach) [reply README.md]: Sounds a bit weird to me too. Maybe "You can use Docker to run your own instance of Servatrice." ?

File: README.md (hunk 3 above)
Comment (TaylorShiells) [reply README.md]: I've removed this entirely based on other suggestions

File: README.md (hunk 2 above)
Comment (TaylorShiells) [reply README.md]: I'm sorry, that was an error on my part. I'm still getting used to github, hopefully I've fixed it.

File: README.md (hunk 6 above)
Comment (TaylorShiells) [reply README.md]: I tried to improve the wording a bit.

File: README.md [hunk 10]
```diff
@@ -2,43 +2,45 @@
...
 
 
 # Downloads [![Cockatrice Eternal Download Count](https://img.shields.io/github/downloads/cockatrice/cockatrice/total.svg)](https://tooomm.github.io/github-release-stats/?username=Cockatrice&repository=Cockatrice)
 
-We offer downloads for all full releases (recommended) and the latest development versions. Full releases are checkpoints with major feature or UI enhancements between them, but currently we don't have a set schedule for releasing new updates. The development version contains the most recently added features and bugfixes, but can be more unstable. Downloads for development versions are updated automatically with every change.
+Downloads are available for all full releases (recommended) and the current development versions. Full releases are checkpoints featuring major feature or UI enhancements. There is no current schedule for future full releases.
+
+The development version contains the most recently added features and bugfixes, but can be unstable. The download for the development version updates automatically with every change.
```
[the remark this answers is not in the document]
Comment (Daenyth) [reply README.md]: You're right; the situation changed and the readme wasn't updated for it. But that's beside the point of this PR

**Comment by Daenyth** [comment]
Thanks very much for the contribution!

File: README.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (tooomm) [reply README.md]: I'll address this in a follow up pr on my own then. 👍

**Comment by ZeldaZach** [comment]
@Daenyth If you wanna merge this, appveyor is hanging

**Comment by Daenyth** [comment]
Done 🎉

**Pull request merged**